pkg/reconciler/revision/resources/pa.go

```diff
@@ -49,6 +50,7 @@ func MakePA(rev *v1alpha1.Revision) *av1alpha1.PodAutoscaler {
 				Name:       names.Deployment(rev),
 			},
 			ProtocolType: rev.GetProtocol(),
+			Reachable:    ptr.Bool(rev.IsReachable()),
```

Comment:
Don't we need to set this to `nil` in certain cases?

Reply:
For the moment, no. It's only nil during upgrades, and the first reconcile will sort it out to either true/false. There may be an edge case that comes out of #4884 but I can include that in that PR.